File: 1672-richest-customer-wealth/1672-richest-customer-wealth.cpp
class Solution {
public:
    int maximumWealth(vector<vector<int>>& accounts) {
        int maxWealth = 0;
        for(vector<int>& account : accounts){
            int currWealth = 0;
            for(int money: account){
                currWealth += money;
            }
            maxWealth = max(maxWealth, currWealth);
        }
        return maxWealth;
    }
};

File: 1672-richest-customer-wealth/1672-richest-customer-wealth.py
class Solution:
    def maximumWealth(self, accounts: List[List[int]]) -> int:
        max_wealth = 0
        curr_wealth = 0
        for customer_account in accounts:
            for a in customer_account:
                curr_wealth += a
            if(curr_wealth > max_wealth):
                max_wealth = curr_wealth
            curr_wealth = 0
        return max_wealth